apps/mini-rx-angular-demo/src/app/modules/todos/state/todos-store.service.ts

```diff
@@ -1,12 +1,18 @@
-import {Injectable} from '@angular/core';
-import {Todo} from '../../todos-shared/models/todo';
-import {TodoFilter} from '../../todos-shared/models/todo-filter';
-import {pipe} from 'rxjs';
-import {mergeMap} from 'rxjs/operators';
-import {v4 as uuid} from 'uuid';
-import {Action, createFeatureSelector, createSelector, FeatureStore, tapResponse,} from 'mini-rx-store';
-import {TodosApiService} from '../../todos-shared/services/todos-api.service';
-import {cloneDeep} from "lodash-es";
+import { Injectable } from '@angular/core';
+import { Todo } from '../../todos-shared/models/todo';
+import { TodoFilter } from '../../todos-shared/models/todo-filter';
+import { pipe } from 'rxjs';
+import { mergeMap } from 'rxjs/operators';
+import { v4 as uuid } from 'uuid';
```

Comment:
Just to note that there is a native method `randomUUID` available in all modern browsers: https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

Reply:
used now crypto in the signal store demo

mini rx store demo needs to be revisited later.